anda/themes/fluent-theme/update.rhai

```diff
@@ -0,0 +1,3 @@
+let release = gh("vinceliuice/Fluent-gtk-theme")
+rpm.global("commit", sha);
```

Comment:
`sha` is undefined?

Reply:
DID YOU FORGET SEMICOLON COME ON LEA WE HAVE THE BEST DEVDOCS IN THE WORLD

Reply:
fuck, I am so out of it tonight